acceptance/config/beaker/jenkins/redhat7-64m-64a.cfg

```diff
@@ -0,0 +1,30 @@
+---
+HOSTS:
+  redhat7-64-1:
+    roles:
+    - agent
+    - master
+    hypervisor: vcloud
```

Comment:
Apologies for the confusion, genconfig2 from 0.13 will produce the correct, non-deprecated output.